app/components/polls/questions/question_component.rb

```diff
@@ -48,28 +48,41 @@ def multiple_choice_help_text
       )
     end
 
-    def multiple_choice_field(option)
-      choice_field(option) do
+    def choice_field(option)
+      label_tag("web_vote_option_#{option.id}") do
+        html = input_tag(option) + option.title
+        html += open_text_tag(option) if option.open_text?
+        html
```

Comment:
According to the W3C validator:

> The label element may contain at most one button, input, meter, output, progress, select, or textarea descendant.

So it looks like we need to move the `open_text_tag` bit outside the `<label>` tag :thinking:.